fennel/datasets/datasets.py

```diff
@@ -2298,6 +2319,21 @@ def visitAggregate(self, obj) -> DSSchema:
         input_schema = self.visit(obj.node)
         keys = {f: input_schema.get_type(f) for f in obj.keys}
         values: Dict[str, Type] = {}
+        if isinstance(obj.along, str):
+            along_type = input_schema.values.get(obj.along)
+            if along_type is None:
+                raise ValueError(
+                    f"Along field `{obj.along}` does not exist in input schema non-key non-timestamp values."
+                )
+            if along_type != datetime.datetime:
+                raise ValueError(
+                    f"Along field `{obj.along}` must be a `timestamp` field."
```

Comment:
`error with along kwarg of aggregate operator: not of type datetime` (note that we don't need this to be the timestamp field, just any datetime field)